package.json

```diff
@@ -63,7 +63,7 @@
     "node-fetch": "^2.6.7",
     "object-hash": "^2.2.0",
     "parse-static-imports": "^1.1.0",
-    "pprof": "^3.2.0",
```

Comment:
Is this change required?

Reply:
No reverted, done for testing and no longer needed.